testing/test_git.py

```diff
@@ -280,7 +280,7 @@ def test_git_archive_export_ignore(wd, monkeypatch):
 
 @pytest.mark.issue(228)
 def test_git_archive_subdirectory(wd, monkeypatch):
-    wd("mkdir foobar")
+    os.mkdir(wd.cwd / "foobar")
```

Comment:
I had to change this as otherwise the test fails with `FileNotFoundError: [WinError 2] The system cannot find the file specified` because it can't find `mkdir`.